File: MyCourseApp/app/src/main/java/com/pkasemer/mycourseapp/MainActivity.java
package com.pkasemer.mycourseapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }


    public void viewCourses(View view){
        String mycourselist = "1. Mobile Programming \n2. Artitificial Intelligence \n3. Network Security and Information \n4. Data communication and Security \n5.Automata and Complexity \n6.Entrepreneurship";
        TextView coursetext = findViewById(R.id.setcourses);
        coursetext.setText(mycourselist);
    }
}

File: MyCourseListKotlin/app/src/main/java/com/pkasemer/mycourselistkotlin/MainActivity.kt
package com.pkasemer.mycourselistkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    val course_outline = "1. Mobile Programming \n2. Automata & Complexity \n3. Data Communication and Security \n4. Network Security and Information\n5. Artificial Intelligence\n6. Entrepreneurship";

    fun viewCourses(view: View) {
        val setcourses: TextView = findViewById(R.id.setcourses) as TextView
        setcourses.text = course_outline
    }
}